data/elevation-and-terrain/index.html

```diff
@@ -79,10 +79,9 @@ <h3 id="LiDAR">LiDAR</h3>
     <p>Light Detection and Ranging (LiDAR) elevation data is an optical remote sensing technology that can measure the
       distance to, or other properties of a target by illuminating the target with light often using pulses from a
       laser. Currently this is the most accurate elevation dataset UGRC has but it is only available for some areas of
-      the state. In addition to the bare-earth DEMs/DTMs and first-return DSMs, most LiDAR data also has point clouds
-      available {% include contact.html subject=page.title contact=site.data.contacts.elevation text='by request from'
-      %} Visit <a href="https://raster.utah.gov/">Raster.utah.gov</a> to explore all available LiDAR collections. For
-      more information about LiDAR visit <a
+      the state. In addition to the bare-earth DEMs/DTMs and first-return DSMs, LAS point clouds are available from
+      [The National Map](https://apps.nationalmap.gov/downloader) {{ contact }} Visit <a href="https://raster.utah.gov/">Raster.utah.gov</a>
```

Comment:
You can't use markdown in an html page...
```suggestion
      <a href="https://apps.nationalmap.gov/downloader">The National Map</a> {{ contact }} Visit <a href="https://raster.utah.gov/">Raster.utah.gov</a>
```